packages/mco/components/modals/app-manage-policies/consistency-groups.tsx

```diff
@@ -0,0 +1,312 @@
+import * as React from 'react';
+import {
+  CONSISTENCY_GROUP_LABEL,
+  LAST_APP_DEPLOYMENT_CLUSTER_ANNOTATION,
+} from '@odf/mco/constants';
+import {
+  ACMManagedClusterViewKind,
+  DRPlacementControlKind,
+  DRVolumeReplicationGroupKind,
+} from '@odf/mco/types';
+import {
+  ACMManagedClusterViewModel,
+  FieldLevelHelp,
+  getAnnotations,
+  getName,
+  getNamespace,
+  GreenCheckCircleIcon,
+  ModalBody,
+  useCustomTranslation,
+} from '@odf/shared';
+import { BLOCK, FILESYSTEM } from '@odf/shared/constants/consistencyGroup';
+import { referenceForModel } from '@odf/shared/utils';
+import { useK8sWatchResources } from '@openshift-console/dynamic-plugin-sdk';
+import {
+  Button,
+  Divider,
+  Flex,
+  FlexItem,
+  MenuToggle,
+  MenuToggleElement,
+  PopoverPosition,
+  SearchInput,
+  Select,
+  SelectList,
+  SelectOption,
+  Text,
+  TextVariants,
+} from '@patternfly/react-core';
+import { FilterIcon } from '@patternfly/react-icons';
+
+const FILTERS = ['all', 'block', 'filesystem'];
+
+export type ConsistencyGroupInfo = {
+  name: string;
+  type: typeof BLOCK | typeof FILESYSTEM;
+  namespace: string;
+  state: string;
+  lastSyncTime: string;
+  pvcs: string[];
+};
+
+export type ConsistencyGroupsContentProps = {
+  consistencyGroups: ConsistencyGroupInfo[];
+  namespace?: string;
+};
+
+const getCGType = (cgName: string) =>
+  cgName.startsWith('cephfs-') ? 'Filesystem' : 'Block';
```

Comment:
utilise the variables which u have already defined...
```suggestion
  cgName.startsWith('cephfs-') ? FILESYSTEM : BLOCK;
```